src/systemTest/java/com/flightstats/hub/kubernetes/PodRestart.java

```diff
@@ -0,0 +1,36 @@
+package com.flightstats.hub.kubernetes;
+
+import io.fabric8.kubernetes.api.model.DoneablePod;
+import io.fabric8.kubernetes.api.model.Pod;
+import io.fabric8.kubernetes.api.model.PodList;
+import io.fabric8.kubernetes.client.DefaultKubernetesClient;
+import io.fabric8.kubernetes.client.dsl.MixedOperation;
+import io.fabric8.kubernetes.client.dsl.PodResource;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.List;
+
+@Slf4j
+public class PodRestart {
+
+    @SneakyThrows
+    public void execute(String releaseName, List<String> podNames) {
+
+        try (final DefaultKubernetesClient client = new DefaultKubernetesClient()) {
```

Comment:
A stray ```final```. Actually, there's a lot of them in a lot of the classes in this PR. I'm guessing it's not really new code, but moved from somewhere?

Reply:
Yes that's me :)

Reply:
I'll go through and clean 'em up.